files/tubular/update.json

```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
+  "name": "tubular",
+  "repo": "unosquare/tubular",
+  "files": {
+    "basePath": "dist/",
+    "include": ["tubular-bundle.js", "tubular-bundle.min.js", "tubular-bundle.css","tubular-bundle.min.css"]
```

Comment:
remove the unminified files and then either squash your commits or create a new PR.
The max amount of commits is 2 per PR